pkg/app/background.go

```diff
@@ -178,29 +181,62 @@ func fileCanBeImported(p string, i os.FileInfo) bool {
 	return false
 }
 
-func (a *App) updateRouteSegments(l *slog.Logger) {
-	var rsID []int
+/** For the given set of route segments, re-match against all workouts, marking the segments as dirty=faslse after matching. */
```

Comment:
Single-line comments should start with `//`; also typo `faslse`; I would suggest to rephrase it: "marking the segments as clean after matching."